english/blackAndwhite/blackAndwhite.md

```diff
@@ -14,6 +14,12 @@ Along the direction of the wind, through the window next to the bed, A tall and
 wearing a leather hood and jeans, smirks seeing Christy tremble in fear.
 
 Christy sprints to the window to check if someone is hiding there and sees no one. Stranger drifts into the eternal dark
-and leaves a puzzle behind for her.
 
-Christy's phone vibrates and drops off the table and Christy screams in dismay waking up the entire dorm.
\ No newline at end of file
+leaving a puzzle behind for her.
+
+Christy's phone vibrates and drops off the table and Christy screams in dismay waking up the entire dorm. 
+
+Everyone wakes up and walk into her messy room trying to figure out what happened. Christy's best friends Bella and Marie console her 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.